guides/plugins/plugins/checkout/document/add-custom-document-type.md

```diff
@@ -339,9 +330,9 @@ Here's what the function does:
 * For each order found, the function attempts to generate a document.
 * If successful, a new `RenderedDocument` object is created. This object is then added to the `RendererResult` object as a success.
 * If an error occurs, the exception is caught and the error message is added to the `RendererResult` object as an error.
-* `The RendererResult` object is returned.
+* The `RendererResult` object is returned.
 
-In this example, we are rendering a specific template, which we will have a look at in the next section.
+Depending on the file type we either get the content with `$this->fileRendererRegistry->render()` or we need to create the content on our own.
```

Comment:
Maybe we can describe better, when we use the `FileRendererRegistry` and when not?